subset/network/network_tests.py

```diff
@@ -0,0 +1,91 @@
+import subprocess
+import time
+import sys
+
+name_of_tests = ['network.min_send', 'network.application.min_send', 'dhcp.long', 'ntp.update']
+
+arguments = sys.argv
+
+test_request = str(arguments[1])
+
+test_id = -1
+
+for x in range(0, len(name_of_tests) - 1):
+    if test_request == name_of_tests[x]:
+        test_id = x
+
+print('test_id=' + str(test_id))
+
+cap_pcap_file = str(arguments[2])
+#capture.pcap
+
+device_address = '127.0.0.1'
+
+if test_id == 0:
+    device_address = str(arguments[3])
+
+report_filename = 'report.txt'
+
+packets_in_report = 10
+
+tcpdump_display_all_packets = 'tcpdump -n src host ' + device_address + ' -r ' + cap_pcap_file
+tcpdump_display_udp_bacnet_packets = 'tcpdump -n udp dst portrange 47808-47809 ' + cap_pcap_file
+tcpdump_display_arp_packets = 'tcpdump -v arp -r ' + cap_pcap_file
+tcpdump_display_ntp_packets = 'tcpdump dst port 123 -r ' + cap_pcap_file
```

Comment:
I don't understand what this is doing. You won't see ntp or any other unicast traffic in this test container (since it's sent only to the destination) -- I'm confused as to what this test is doing.

Reply:
This code is for use with the other Docker build file added to this PR. Using the code this way with multiple docker containers allows separate tests to pass as a line item in DAQ feedback page
https://curious-song-211910.firebaseapp.com/?origin=daq-testing-account

Reply:
That didn't answer my question.  You can't/shouldn't be capturing traffic not intended for this container within this container, since you likely will never see it. NTP is going to unicast to the NTP server, not this container. And even things like arp aren't guaranteed to make it here if the networking system decides they don't need to (ARP is a very low-level networking operation).

This is why the scans/ folder exists, since they are drawn from content that's outside of the container.